bankfind/base.py

```diff
@@ -121,6 +129,7 @@ def _get_data(
             search: str = None,
             **kwargs):
         params = self._construct_params(key, filters, search, **kwargs)
+        print(urllib.parse.urlencode(params))
```

Comment:
Do you really mean to leave this print statement in here?  If this project already has a logging facility it might be best to use that or hide this print statement behind a debug environment variable or something.